tests/unit_tests/test_workflow.py

```diff
@@ -296,3 +297,51 @@ def test_workflow_base():
             # os.rmdir(os.path.join(my_workflow.path, my_workflow.FIGURE_PATH))
             # os.rmdir(os.path.join(my_workflow.path))
             shutil.rmtree(os.path.join(my_workflow.parent_path))
+
+
+FDR_TEST_BASE_CLASSIFIER = BinaryClassifierLegacyNewBatching(
+    test_size=0.001, batch_size=50, learning_rate=0.001, epochs=1
+)
+FDR_TEST_FEATURES = ["feature_a", "feature_b"]
+
+
+def fdr_testdata(features):
+    test_dict = {}
+
+    for feature in features:
+        test_dict[feature] = np.random.normal(50, 2, 100)
+    test_dict["decoy"] = np.random.randint(0, 2, 100)
+    test_dict["precursor_idx"] = np.random.randint(1000, 10000, 100)
+
+    return pd.DataFrame(test_dict)
+
+
+def test_fdr_manager():
+    fdr_manager = manager.FDRManager(FDR_TEST_FEATURES, FDR_TEST_BASE_CLASSIFIER)
+
+    assert fdr_manager.is_loaded_from_file is False
+    assert fdr_manager.is_fitted is False
+
+    assert fdr_manager.feature_columns == FDR_TEST_FEATURES
+    assert fdr_manager.classifier_base == FDR_TEST_BASE_CLASSIFIER
+
+
+@pytest.mark.slow
```

Comment:
Not necessary. You should be able to replace `BinaryClassifierLegacyNewBatching` with any scikit learn classifier (or even a mock) to speed it up.

Reply:
The FDRManager.fit_predict() method requires the classifier to have the method .from_state_dict().

Reply:
Is there an alternative way to speed this up? Or is it better to leave this as it is?